src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsConfig.java

```diff
@@ -72,28 +73,44 @@
     /**
      * Users can define name/password pairs at the global level to share common
      * passwords with several jobs.
-     * 
+     *
      * <p>Never ever use this attribute directly: Use {@link #getGlobalVarPasswordPairsList} to avoid
      * potential NPEs.</p>
-     * 
+     *
      * @since 2.7
      */
     private List<VarPasswordPair> globalVarPasswordPairs;
+    /**
+     * Users can define regexes at the global level to mask in jobs.
+     *
+     * <p>Never ever use this attribute directly: Use {@link #getGlobalVarPasswordPairsList} to avoid
+     * potential NPEs.</p>
+     *
+     * @since 2.9
+     */
+    private List<VarMaskRegex> globalVarMaskRegexes;
```

Comment:
Well, I agree there is no direct usages of globalVarMaskRegexes excepting the getter list, hence it should be fine